### PR TITLE
[PPML] Decrease Python Version To 3.7 And Install TF1 For Graphene

### PR DESCRIPTION
**Pull request by piaolaidelangman**
* Decrease python version to 3.7.
* Change python version to 3.7 in `bash.manifest.template`.
* Install tensorflow==1.15.
* Delete `sudo` in azure cli install.
* The examples in README and wiki have been tested.

### CODE REVIEW

**Comment by glorysdj** [comment]
please test this in local container first.

File: ppml/trusted-big-data-ml/python/docker-graphene/Makefile [hunk 1]
```diff
@@ -27,7 +27,6 @@ all: bash.token
 endif
 
 include $(GRAPHENEDIR)/Scripts/Makefile.configs
-PYTHONVERSION = python3.6
```
Comment (qiyuangong) [inline ppml/trusted-big-data-ml/python/docker-graphene/Makefile:30]: Do we need to change to python 3.7?

Comment (piaolaidelangman) [reply ppml/trusted-big-data-ml/python/docker-graphene/Makefile:30]: It doesnot work. I think we can just delete it.

**Review by qiyuangong — APPROVED** [review]
LGTM

**Comment by piaolaidelangman** [comment]
The examples in README and wiki have been tested.

**Comment by qiyuangong** [comment]
This PR is not just for PPML. It may also work for orca.

**Review by glorysdj — APPROVED** [review]
LGTM

**Pull request merged**